lib/Backend/arm64/LowerMD.cpp

```diff
@@ -865,18 +849,21 @@ LowererMD::GenerateStackProbe(IR::Instr *insertInstr, bool afterProlog)
     IR::LabelInstr *doneLabelInstr = IR::LabelInstr::New(Js::OpCode::Label, this->m_func, false);
     if (!IS_FAULTINJECT_STACK_PROBE_ON) // Do stack check fastpath only if not doing StackProbe fault injection
     {
+        // CMP sp, r17 
         instr = IR::Instr::New(Js::OpCode::CMP, this->m_func);
         instr->SetSrc1(IR::RegOpnd::New(nullptr, GetRegStackPointer(), TyMachReg, this->m_func));
         instr->SetSrc2(scratchOpnd);
         insertInstr->InsertBefore(instr);
         LegalizeMD::LegalizeInstr(instr, false);
 
+        // BGT  done
         instr = IR::BranchInstr::New(Js::OpCode::BGT, doneLabelInstr, this->m_func);
```

Comment:
It doesn't matter in practice, but this should be an unsigned compare and thus the branch should be BHI instead of BGT.

Reply:
Updated to BHI

---
In reply to: [144412622](https://github.com/Microsoft/ChakraCore/pull/3943#discussion_r144412622) [](ancestors = 144412622)